package.json

```diff
@@ -127,11 +127,14 @@
       "text",
       "html",
       "lcov"
-    ]
+    ],
+    "moduleNameMapper": {
+      "axios": "axios/dist/node/axios.cjs"
```

Comment:
Just wanted to understand why we did this. We can discuss this over call. Otherwise everything is working fine.

Reply:
This is a workaround to fix unit test cases related to Axios code. See: https://github.com/axios/axios/issues/5101